src/modelfree/score_agent.py

```diff
@@ -49,6 +54,37 @@ def get_empirical_score(_run, env, agents, episodes, render=False):
 
     return result
 
+def _clean_video_directory_structure(observer_obj):
+    """
+    A simple utility method to take saved videos within a Sacred run structure and clean up the file pathways, so that
+    all videos are organized under a "videos" directory, and within that organized by environment ID.
+
+    :param observer_obj: A Sacred FileStorageObserver object
+    :return:
+    """
+    basedir = observer_obj.dir
+    video_files = glob.glob("{}/*.mp4".format(basedir))
+    if len(video_files) == 0:
+        return
+
+    ptn = re.compile('env_(\d*)_video.(\d*)')
```

Comment:
I think this should be using Python's raw string syntax: r''. Otherwise the `\d` could be interpreted as escape sequences.